src/implementations/vanilla/utilities/spacing/spacing.scss

```diff
@@ -1,5 +1,5 @@
 @use 'sass:map';
-@use '@ecl/theme-dev/theme';
+@use '@ecl/theme-ec/theme';
```

Comment:
Mmhh..this is confusing, might be better to first finalise the other pr and integrate it here in the utilities.